_dev/cldr/number-formatter.js

```diff
@@ -270,11 +276,24 @@ class CurrencyFormatter {
         specifications.secondaryGroupSize,
         specifications.currencySymbol,
         specifications.currencyCode,
-      ),
-    );
+      );
+    } else {
+      specification = new NumberSpecification(
+        specifications.positivePattern,
+        specifications.negativePattern,
+        symbol,
+        parseInt(specifications.maxFractionDigits, 10),
+        parseInt(specifications.minFractionDigits, 10),
+        specifications.groupingUsed,
+        specifications.primaryGroupSize,
+        specifications.secondaryGroupSize,
+      );
+    }
+
+    const currency = new NumberFormatter(specification);
```

Comment:
Maybe you should name it `currency` if it's for generic now

Reply:
you're right!